client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -1,452 +1,122 @@
-                      onBlur={save}
-                    />
-                  }
-                  labelWidth={'66px'}
-                  label={t('label.reason')}
-                  sx={{ marginBottom: 1 }}
-                />
-              )}
-              <InputWithLabelRow
-                Input={
-                  <TextArea
-                    value={draft?.comment ?? ''}
-                    onChange={e => update({ comment: e.target.value })}
-                    slotProps={{
-                      input: {
-                        sx: {
-                          backgroundColor: theme =>
-                            theme.palette.background.menu,
-                        },
-                      },
-                    }}
-                    onBlur={save}
-                    disabled={isSent}
-                  />
-                }
-                sx={{ width: 275 }}
-                labelWidth={LABEL_WIDTH}
-                label={t('label.comment')}
-              />
-            </Box>
-          </>
-        )}
-      </Box>
+  useEffect(() => {
+    if (data) setCurrentItem(data);
+  }, [data, setCurrentItem]);
 
+  return (
+    <Box display="flex" flexDirection="column" padding={2} gap={1}>
+      <AccordionPanelSection
+        key={`${key}_item_search`}
+        title={t('label.item', { count: 1 })}
+        closedSummary={draft?.itemName}
+        defaultExpanded={!disabled}
+      >
+        <StockItemSearchInput
+          currentItemId={draft?.itemId}
+          onChange={handleChange}
+          disabled={disabled}
+          extraFilter={item => !lines.some(line => line.item.id === item.id)}
+        />
+      </AccordionPanelSection>
+      <AccordionPanelSection
+        key={`${key}_order`}
+        title={t('title.order')}
+        defaultExpanded={!isNew && !disabled}
+      >
+        <Order
+          disabled={disabled}
+          isPacksEnabled={isPacksEnabled}
+          draft={draft}
+          update={update}
+        />
+      </AccordionPanelSection>
+      <AccordionPanelSection key={`${key}_details`} title={t('label.details')}>
+        <Details
+          isProgram={isProgram}
+          draft={draft}
+          update={update}
+          plugins={plugins}
+          isPacksEnabled={isPacksEnabled}
+          disabled={disabled}
+        />
+      </AccordionPanelSection>
+      <AccordionPanelSection
+        key={`${key}_comment`}
+        title={t('label.comment')}
+        defaultExpanded={false}
+      >
+        <TextArea
+          value={draft?.comment ?? ''}
+          onChange={e => update({ comment: e.target.value })}
+          disabled={disabled}
+        />
+      </AccordionPanelSection>
       <Box paddingTop={1} maxHeight={200} width={width * 0.48} display="flex">
         {line &&
           plugins.requestRequisitionLine?.editViewInfo?.map((Info, index) => (
             <Info key={index} line={line} requisition={requisition} />
           ))}
       </Box>
```

Comment:
I think this is a wait for feedback thing, but I wonder if plugin info would get lost down here below charts and comment? Guess it depends on how critical said plugin data is for user... but now that we're in a modal, you won't see it without scrolling

Reply:
We could move it to its own accordion as well? That being said I'll have to set up plugins for my environment. But that shouldn't be too bad.